docs/en/developers/03_Backend/05_Extensions.md

```diff
@@ -179,7 +206,21 @@ Minz_Request::bad($feedback_bad, $url_array);
 
 ### Translation Management
 
-**TODO** translate from [french version](https://github.com/FreshRSS/documentation/blob/master/fr/docs/developers/03_Backend/05_Extensions.md)
+It is common (and that's an understatement) to want to show some text to the user. In the previous example, for example, we display feedback to the user based on the result of form validation. The problem is that FreshRSS has users of different nationalities. It is therefore necessary to be able to manage different languages in order not to remain confined to English or French.
+
+The solution is to use the `Minz_Translate` class, which allows dynamic translation of FreshRSS (or any Minz-based application). Before using this module, it is necessary to know where to find the strings to be translated. Each language has its own subdirectory in a parent directory named `i18n`. For example, English language files are located in `i18n/fr/`. There are seven different files:
+
+- `admin.php` for anything related to FreshRSS administration
+- `conf.php` for configuration
+- `feedback.php` contains translations of feedback messages
+- `gen.php` stores what is global to FreshRSS (gen for "general")
+- `index.php` for the main page that lists feeds and the About page
+- `install.php` contains strings related FreshRSS installation
+- `sub.php` for subscription management (sub for :subscription")
```

Comment:
Drat, missed this typo. (Note to self.)

Reply:
Arf, new PR :-)

Reply:
If you send a correction, I will merge to /master without making a new release (as this is only the documentation). I will wait for that before starting 1.9.1-dev.

Reply:
Unless you'd like to get it fixed ASAP I'll just include it in an upcoming PR for some style/grammar fixes in other parts of the docs. :-)

Reply:
Then again I can just do it from GH itself in two secs.